quaint/src/connector/mysql/url.rs

```diff
@@ -57,8 +57,8 @@ impl MysqlUrl {
     /// Name of the database connected. Defaults to `mysql`.
     pub fn dbname(&self) -> &str {
         match self.url.path_segments() {
-            Some(mut segments) => segments.next().unwrap_or("mysql"),
-            None => "mysql",
+            Some(mut segments) => segments.next().unwrap_or(super::defaults::DEFAULT_MYSQL_DB),
+            None => super::defaults::DEFAULT_MYSQL_DB,
         }
```

Comment:
Unrelated: Is defaulting to `mysql` really a good idea usually? It's a system database...

Reply:
Yea, it's probably not. I've just copied what our native URLs parsing code did, but I don't think it has much sense. Might be worth looking into what other drivers do, probably we should throw if it's not specified.

Reply:
Definitely something we need to tread carefully. 
But I wonder what that even affects...